src/cli/config.js

```diff
@@ -94,7 +98,16 @@ async function getConfigurationFile(filename, folder) {
         }
     }
 
-    return await getDefaultConfigurationFileTraversingParents(folder);
+    // Look at parent directories or, finally, the user's home directory
```

Comment:
Could you remove this comment, I think the code is rather explicit about what is happening already.

As an additional suggestion, to make to code (without comment) even more explicit: you could extract the homedir config file-related logic into a separate function and call+return that conditional on whether or not `getDefaultConfigurationFileTraversingParents` returns something.

This could look something like

```js
filepath = await getDefaultConfigurationFileTraversingParents(folder);
if (filepath) {
  return filepath;
}

return await getConfigurationInHomedir();
```

Reply:
Removed.

Reply:
It appears the comment is still there.

Reply:
My mistake.